sdk/communication/AzureCommunicationChat/x86_64.swiftinterface

```diff
@@ -0,0 +1,543 @@
+// swift-interface-format-version: 1.0
+// swift-compiler-version: Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)
+// swift-module-flags: -target x86_64-apple-ios12.0-simulator -enable-objc-interop -enable-library-evolution -swift-version 5 -enforce-exclusivity=checked -Onone -module-name AzureCommunicationChat
+import AzureCommunication
+import AzureCore
+import Foundation
+import Swift
+public class ChatClient {
```

Comment:
Some of these are generated, calling out the convenience layer specifically for review